packages/create-instance/create-component-stubs.js

```diff
@@ -75,6 +81,24 @@ function resolveOptions(component, _Vue) {
   return options
 }
 
+function getScopedSlotRenderFunctions(ctx: any): Array<string> {
+  // In Vue 2.6+ a new v-slot syntax was introduced
+  // scopedSlots are now saved in parent._vnode.data.scopedSlots
+  // We filter out the _normalized and $stable key
+  if (
+    ctx &&
+    ctx.$options &&
+    ctx.$options.parent._vnode &&
+    ctx.$options.parent._vnode.data &&
+    ctx.$options.parent._vnode.data.scopedSlots
+  ) {
+    const slotKeys: Array<string> = ctx.$options.parent._vnode.data.scopedSlots
+    return keys(slotKeys).filter(x => x[0] !== '_' && !x.includes('$'))
```

Comment:
Thinking out loud here, and might be missing something, but:

If we're aiming to remove `_normalized` and `$stable`,  could we be more explicit about our intention?

```js
return keys(slotKeys).filter(x => x !== '_normalized' && x !== '$stable')
```

(or even `['_normalized', '$stable'].includes(x)`)

Reply:
Yep, sounds good. I don't think there should be any other special variables beginning with `_` or `$`.